src/components/Card/Card.spec.js

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@testing-library/react'
+import Card from './Card'
+
+describe('Card', () => {
+  it('renders title and image of Elbphilharmonie', () => {
```

Comment:
the title says that it should test for an image, either change the title or add a test for an image (which I would prefer 😉)